docs/docs/configuration/index.html

```diff
@@ -1009,16 +1009,17 @@ <h3 id="values">Values</h3>
 <td>string</td>
 <td>Pagure token needed to access REST API, get it at: <a href="https://src.fedoraproject.org/settings#nav-api-tab">https://src.fedoraproject.org/settings#nav-api-tab</a></td>
 </tr>
-
-<tr>
-<td><code>pagure_fork_token</code></td>
-<td>string</td>
-<td>a token so packit can create a pull request: <a href="https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab">https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab</a></td>
-</tr>
 </tbody>
 </table>
 
-<p>You can also specify the tokens as environment variables: <code>GITHUB_TOKEN</code>, <code>PAGURE_USER_TOKEN</code>, <code>PAGURE_FORK_TOKEN</code>.</p>
+<p>You can also specify the tokens as environment variables: <code>GITHUB_TOKEN</code>,
+<code>PAGURE_USER_TOKEN</code>.</p>
```

Comment:
Is this still relevant after today arch meeting? You are right that it is still there, but we are going to deprecate it soon anyway.